src/Umbraco.AuthorizedServices/Umbraco.AuthorizedServices.csproj

```diff
@@ -27,11 +27,16 @@
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
+  <!-- Umbraco packages -->
+  <PropertyGroup>
+    <UmbracoCmsPackageVersion>[15.0.0, 16)</UmbracoCmsPackageVersion>
```

Comment:
Lets go for `[15.0.0, 15.999.999]` here.  We've discussed this might be better as it makes explicit that `16.0.0-rc1` won't run on CMS `15,0`, whereas the syntax you (and we, traditionally) have used would.